packages/fx-core/src/component/generator/copilotPlugin/helper.ts

```diff
@@ -241,17 +241,40 @@ export async function listOperations(
       const manifest = await manifestUtils._readAppManifest(teamsManifestPath);
       let existingOperations: string[] = [];
       if (manifest.isOk()) {
+        const operationIds = operations.map((operation) => operation.api);
```

Comment:
might be better to rename this to something like "operationApis"? operationIds is kind of confusing here.